Gemfile

```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+git_source(:github){ |repo_name| "https://github.com/#{repo_name}" } if respond_to?(:git_source)
```

Comment:
The `:github` shortcut is included with bundler (as is `:gist` and `:bitbucket`), so defining a custom source for it isn't necessary.